data/chapter-to-page-mappings.json

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
Two issues here:

1. This mapping will only work for Mushafs with 604 pages. Indopak Mushafs for example don't have 604 pages.
2. We already have an API to get information about pages (`/pages/lookup`) which takes current mushafId + chapterNumber and returns which pages includes in the current chapter while respecting the mushaf.